src/github.com/getlantern/fdcount/fdcount.go

```diff
@@ -41,16 +40,23 @@ func (c *Counter) AssertDelta(expected int) error {
 	if err != nil {
 		return err
 	}
-	actual := c.countMatches(out) - c.startingCount
+	endingLines, endingCount := c.matchingLines(out)
```

Comment:
fdcount is a utility for counting file descriptors.  I made some changes to make the messages that it reports more useful.